lib/voice/VoiceConnection.js

```diff
@@ -738,6 +879,15 @@ class VoiceConnection extends EventEmitter {
                 }
             }
 
+            const userID = this.ssrcUserMap[msg.readUIntBE(8, 4)];
+            if(this.#daveReady() && userID && data !== SILENCE_FRAME) {
```

Comment:
this will always fail as data will never be referentially equal to `SILENCE_FRAME`, compare buffer contents instead

```suggestion
            if(this.#daveReady() && userID && !data.equals(SILENCE_FRAME)) {
```